aws-serverless-java-container-core/src/main/java/com/amazonaws/serverless/proxy/internal/servlet/AwsProxyHttpServletRequest.java

```diff
@@ -471,6 +480,12 @@ public RequestDispatcher getRequestDispatcher(String s) {
 
     @Override
     public int getRemotePort() {
+        if (Objects.nonNull(request.getRequestContext().getElb())) {
+            String hostHeader = request.getHeaders().get(HttpHeaders.HOST);
+            String port = Arrays.asList(hostHeader.split(":")).get(1);
```

Comment:
I'm currently seeing `java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1` when calling `getRemotePort()`.

For a http Request on 80 the payload looks like this:

```
{
    "requestContext": {
        "elb": {
            "targetGroupArn": "arn:aws:elasticloadbalancing:eu-central-1:5952169XXXXX:targetgroup/sam-ap-MyTar-ZEEXXXXX/77425e11d8XXXX"
        }
    },
    "httpMethod": "GET",
    "path": "/",
    "queryStringParameters": {},
    "headers": {
        "accept": "*/*",
        "host": "sam-ap-MyLoa-jNFZXXXXX-000000000.eu-central-1.elb.amazonaws.com",
        "user-agent": "curl/8.4.0",
        "x-amzn-trace-id": "Root=1-661944f2-348171d959c3a5cb69d00000",
        "x-forwarded-for": "1.2.3.4",
        "x-forwarded-port": "80",
        "x-forwarded-proto": "http"
    },
    "body": "",
    "isBase64Encoded": false
}
```